src/screens/ResultScreen.js

```diff
@@ -1,18 +1,24 @@
 import React from 'react'
 import { StyleSheet, View, Text, Button, Image, ScrollView } from 'react-native'
-
+import openMap from 'react-native-open-maps';
 
 const ResultScreen = () => {
+  const goToRestaurant = () => {
+    openMap({ provider: Platform.OS === 'ios' ? 'apple':'google', start: 'my location', travelType: 'walk', end: 'Casa Bonita'  });
```

Comment:
Are these the maps/directions the user should see after they submit preferences? I see it's set for 'ios'/'apple', so does it mean it's only set for iPhones at the moment?

Reply:
I was wondering the same, or is this a ternary operator?